### DISCUSSION TITLE
Change/Create Feeding Amount Graph to Match Style Pumping Amount

### DISCUSSION THREAD

**Discussion by rwelsh09** [babybuddy/babybuddy#858] [Q&A]
Is it possible to change the feeding amount graph to look like the pumping amount graph? (stacked bar graph with each amount a different colour)

I thought I might be able to create this using the [plotly graph card](https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/434), but I am quite lost.

Thanks

**Comment by cdubz** [comment]
Feeding amounts are stacked by type currently. You're interested in stacking by feeding amount?

**Reply by rwelsh09** [reply]
Sorry no, not by amount but by session like the pumping graph, the first pump session is at the bottom and each subsequent session is stacked on top. Each session is a different color and it resets daily.

**Reply by cdubz** [reply]
Ok, I see what you are saying. I wouldn't necessarily want to change the behavior of the current graph, but we could add another one. Just might be awkward to figure out naming, hah.

**Reply by rwelsh09** [reply]
Thanks for considering the feature! I think adding a separate graph with the suggested stacking method would be really useful. Perhaps it could be named 'Feeding Sessions' to clearly differentiate it?
